Boot2Docker Status/AppDelegate.m

```diff
@@ -172,11 +180,24 @@ - (void)stopBoot2Docker {
     
     [task setTerminationHandler:^(NSTask *t) {
         _haveRequestedStateChange = NO;
-        _statusItem.image = [NSImage imageNamed:@"docker-alt"];
+		_statusItem.image = [self _iconWithNotificationBubbleWithColor:[NSColor redColor]];
     }];
     
     [task launch];
 }
 
+- (NSImage*)_iconWithNotificationBubbleWithColor:(NSColor*)color {
+	NSImage* image = [(_darkMode ? [NSImage imageNamed:@"docker-dark"] : [NSImage imageNamed:@"docker"]) copy];
+
```

Comment:
why do we need the alt image? when boot2docker is down we just put a red dot, right?